tests/e2e/etcd_process.go

```diff
@@ -105,6 +125,15 @@ func (ep *etcdServerProcess) Start() error {
 	if ep.proc != nil {
 		panic("already started")
 	}
+	if ep.cfg.proxy != nil && ep.proxy == nil {
+		// ep.cfg.lg.Info("starting proxy...", zap.String("name", ep.cfg.name), zap.String("from", ep.cfg.proxy.From.String()), zap.String("to", ep.cfg.proxy.To.String()))
```

Comment:
Any reason to leave this here?

Reply:
there is no log in 3.4 implementation of etcd process. Removed the line.